app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<form class="form">
+  <h2 class="heading">Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field input-div">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field input-div">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
```

Comment:
- [ ] after regristation i could not login in kindly check this [video](https://www.loom.com/share/1bc7e689f2934eb78c41f6123beed6d9) out